fenix/app/src/main/java/org/mozilla/fenix/compose/Image.kt

```diff
@@ -46,9 +49,13 @@ fun Image(
     contentDescription: String? = null,
     alignment: Alignment = Alignment.Center,
     contentScale: ContentScale = ContentScale.Fit,
+    placeholder: Painter = ColorPainter(FirefoxTheme.colors.layer3),
```

Comment:
`Painter` is not marked as stable by the compose compiler. That means it will recompose even if the inputs have not changed for this function. 

I think a `placeholder: @Composable () -> Unit` would be good here with the default set to the existing `DefaultPlaceholder` composable